src/index.html

```diff
@@ -10,13 +10,105 @@
       http-equiv="X-UA-Compatible"
       content="ie=edge"
     />
+    <link
+      rel="preconnect"
+      href="https://fonts.googleapis.com"
+    />
+    <link
+      rel="preconnect"
+      href="https://fonts.gstatic.com"
+    />
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500&display=swap"
+      rel="stylesheet"
+    />
     <title>Moyo header</title>
     <link
       rel="stylesheet"
       href="./style.css"
     />
   </head>
+
   <body>
-    <h1>Moyo header</h1>
+    <header class="header">
+      <div class="header__container container">
+        <a href="#">
+          <img
+            src="/src/images/logo.png"
+            alt="logo"
```

Comment:
```suggestion
            alt="Moyo logo"
```
![image](https://github.com/mate-academy/layout_moyo-header/assets/54860641/efd4856a-f692-430b-a36b-4ce14b414604)